doc/about/roadmap.md

```diff
@@ -0,0 +1,44 @@
+# HoloViz Roadmap, as of 11/2024
+
+HoloViz helps coordinate between numerous independent open-source projects, each with its own developers, priorities, and agendas. The future of HoloViz largely depends on a distributed team of developers who focus on the areas of greatest current interest and need, including areas that receive specific funding.
+
+Currently ongoing or scheduled priorities include:
+
+1. **Ongoing maintenance, improved documentation and examples**: Address various bugs and usability issues reported on the issue tracker as time permits.
+
+2. **Better Datashader integration with external plotting libraries (HoloViews, Plotly, Matplotlib)**: Improve integration so external plotting programs can support user-friendly features like hover information, legends, and colorbars.
+
+3. **Support for managing annotations**: Develop annotations as a first-class component in HoloViz tools, allowing users to add annotations to plots and dashboards. See [holonote on GitHub](https://github.com/holoviz/holonote) for progress.
```

Comment:
Is the holonote repo or the project https://github.com/orgs/holoviz/projects/11 the best place to check for progress? In general I'd expect a project, since this is holoviz.org and not a specific project's roadmap, but whichever one is the most suitable is what to list.